### PR TITLE
Remove return

### PR DESCRIPTION
**Pull request by Incompleteusern**
Don't know how this got in there

### CODE REVIEW

File: src/main/java/com/wynntils/modules/map/overlays/ui/SeaskipperWorldMapUI.java [hunk 1]
```diff
@@ -154,7 +154,6 @@ private void updateItems() {
                 location.setCost(cost);
                 location.setActiveType(SeaskipperLocation.Accessibility.ACCESSIBLE);
                 routes.put(location, s.slotNumber);
-                return;
```
Comment (HeyZeer0) [inline src/main/java/com/wynntils/modules/map/overlays/ui/SeaskipperWorldMapUI.java:157]: Should be a continue no?

**Review by HeyZeer0 — CHANGES_REQUESTED** [review]

File: src/main/java/com/wynntils/modules/map/overlays/ui/SeaskipperWorldMapUI.java (hunk 1 above)
Comment (HighCrit) [reply src/main/java/com/wynntils/modules/map/overlays/ui/SeaskipperWorldMapUI.java:157]: Or else if with next statement

Comment (Incompleteusern) [reply src/main/java/com/wynntils/modules/map/overlays/ui/SeaskipperWorldMapUI.java:157]: It doesn't matter if its a continue statement, I like at as an else if but https://github.com/Wynntils/Wynntils/pull/341#discussion_r693610062

Comment (Incompleteusern) [reply src/main/java/com/wynntils/modules/map/overlays/ui/SeaskipperWorldMapUI.java:157]: It's just important that this bug gets fixed, fine either way

**Comment by Incompleteusern** [comment]
Fixed in https://github.com/Wynntils/Wynntils/commit/863e7d97cd2d79cb03eea8d4aafb8f799d52e03d

**Pull request closed without merging**